server/darkmoon/settings.py

```diff
@@ -23,11 +22,11 @@
 class Settings(BaseSettings):
     """Settings class."""
 
-    scheme = str(os.getenv("SCHEME"))
-    ipv4 = str(os.getenv("IP_ADDRESS"))
-    port = str(os.getenv("PORT"))
-    user = str(os.getenv("DATABASE_USERNAME"))
-    password = str(os.getenv("DATABASE_PASSWORD"))
+    SCHEME: str
+    IP_ADDRESS: str
+    PORT: str
```

Comment:
If you don't want to use the Pydantic built-in support for MongoDB DSNs, then the `port` field should be `PORT: int` and the IP address should use the python IPAddress built-in object (Pydantic has a special one you can use as well).

Reply:
The changes have been made! I imported the ipaddress library into settings.py and set IP_ADDRESS to IPv4Address type and PORT to int. These changes are also reflected in the TEMPLATE.env and personal .env files.